app/main/views/buyers.py

```diff
@@ -268,11 +268,27 @@ def find_team_by_team_id():
     team = team_info.get('team')
     briefs = team_info.get('briefs')
 
+    team_leads = team.get('teamLeads')
+    team_leads_email_name = []
+
+    for i in team_leads:
+        team_leads_email_name.append(team_leads.get(i))
+
+    team_members = team.get('teamMembers')
+
+    team_members_email_name = []
+    team_members_permissions = []
```

Comment:
Doesn't look like this is being used.

Reply:
Good catch! - It is just team_members_permissions not being used

Removed now